src/main/java/com/scottlogic/competitionEngine/application/user/UserServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package com.scottlogic.competitionEngine.application.user;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/users")
+public class UserServiceImpl {
```

Comment:
please have this implement the UserService interface which for now can have a single method 'getCurrentUser' returning a String. We will expand this in future iterations.

Reply:
Shouldn't that go to the feature/user-service branch / commit? I've left it out of this one as wasn't sure if it was related / relevant: [link to Trello card](https://trello.com/c/70VcVOE2/10-5-back-end-to-have-user-servicel)

Reply:
i.e., the interface, not the name of the method.

Reply:
I the tickets aren't that clearly defined anyway so it doesnt really matter. If it were me then Id create it here since we have created the service and called it 'Impl' even though it doesn't implement anything at the moment.

Reply:
Ok - fair point about the naming. However, I'll add it in in the next commit as it'll make it a bit easier for me to switch between branches :)